### PR TITLE
Challenge3

### PR DESCRIPTION
**Pull request by DoomStone**
Pull request @callebjorkell and @jesperterkelsen

screenshot 2015-03-31 14 30 31

### CODE REVIEW

File: src/test/java/com/tradeshfit/controllers/HomeControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import org.springframework.ui.ModelMap;
+import org.springframework.web.servlet.ModelAndView;
+
+import static org.mockito.Mockito.mock;
+
+public class HomeControllerTest {
+
+	@Test
+	public void TestConstructor(){
+		new HomeController();
```
Comment (callebjorkell) [inline src/test/java/com/tradeshfit/controllers/HomeControllerTest.java]: What does this test? I smell fake line coverage :smile:

File: src/test/java/com/tradeshfit/controllers/HomeControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import com.tradeshift.controllers.HomeController;
+import org.junit.Assert;
+import org.junit.Test;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.servlet.ModelAndView;
+
+import static org.mockito.Mockito.mock;
+
+public class HomeControllerTest {
```
Comment (callebjorkell) [inline src/test/java/com/tradeshfit/controllers/HomeControllerTest.java:11]: Method naming conventions in this class.

File: src/main/java/com/tradeshift/controllers/HomeController.java [hunk 3]
```diff
@@ -1,7 +1,17 @@
...
+import org.springframework.web.servlet.ModelAndView;
 
+
+@Controller
+@RequestMapping("/")
 public class HomeController {
 
-    public void Index(){
+	@RequestMapping(method = RequestMethod.GET)
+    public ModelAndView Index(){
```
Comment (callebjorkell) [inline src/main/java/com/tradeshift/controllers/HomeController.java]: naming.

**Comment by callebjorkell** [comment]
Could you post a video/other demonstating the new UI? You can use quicktime for easy screen capture.(https://support.apple.com/kb/PH5882?locale=en_US)

**Comment by DoomStone** [comment]
@callebjorkell Video: https://www.youtube.com/watch?v=h7bDehQkVis&feature=youtu.be

**Comment by callebjorkell** [comment]
image

**Comment by callebjorkell** [comment]
You can also post the video on box, or similar.

**Comment by DoomStone** [comment]
The link should work now

**Comment by callebjorkell** [comment]
very nice. :+1:

**Pull request merged**